unicon_backend/models/organisation.py

```diff
@@ -36,6 +36,30 @@ class Project(ProjectBase, table=True):
     problems: sa_orm.Mapped[list["ProblemORM"]] = Relationship(
         back_populates="project", sa_relationship_kwargs={"order_by": "ProblemORM.id.desc()"}
     )
+    groups: sa_orm.Mapped[list["Group"]] = Relationship(back_populates="project")
+
+
+class Group(CustomSQLModel, table=True):
+    """This is closer to tutorial groups and not intended to be used for submitting problems."""
+
+    __tablename__ = "group"
+
+    id: int | None = Field(default=None, primary_key=True)
+    project_id: int | None = Field(foreign_key="project.id")
+
+    name: str
+
+    members: sa_orm.Mapped[list["GroupMember"]] = Relationship(
+        back_populates="group",
+        sa_relationship_kwargs={
+            "order_by": "GroupMember.is_supervisor.desc()",
+            # This line is used to fix an error with updating group members.
```

Comment:
just curious, what is the error and how does `order_by` affect things?

Reply:
it was the "cascade" below that helped! (when i update group members i think i am recreating all rows, including deleting the old ones)